html-css/override-extend-variables/step2-instructions.md

```diff
@@ -1,32 +1,32 @@
-## Override PatternFly variables to achieve a new theme
-
-Now that we have a card on our page, let's customize the styles applied to it.
-
-1) <strong>Add a new color variable to use in the card.</strong> Let's make the links in the card purple! In PatternFly, all of the css values we use are first assigned to variables. We recommend doing the same when making customizations in your application.
-
+## Override PatternFly variables to achieve a new theme
+ 
+Now that you have a card on your page, let's customize the styles applied to it.
+ 
+1) <strong>Add a new color variable to use in the card.</strong> Let's make the links in the card purple! In PatternFly, all of the css values you use are first assigned to variables. It is recommended doing the same when making customizations in your application.
```

Comment:
Add a return statement after the first sentence. 
Remove "you"
`It is recommended doing the same when making customizations in your application.` --> change this sentence it doesn't sound like English